packages/notification-config-process/Dockerfile

```diff
@@ -0,0 +1,60 @@
+#default value for compatibility
+ARG NODE_REGISTRY="docker.io"
+
+FROM ${NODE_REGISTRY}/node:20.14.0-slim@sha256:5e8ac65a0231d76a388683d07ca36a9769ab019a85d85169fe28e206f7a3208e as build
+
+RUN npm install -g corepack@0.31.0 && corepack enable
```

Comment:
Apply fixes from https://github.com/pagopa/interop-be-monorepo/pull/2107, it will be probably merged before

Reply:
Applied in 91cceb5e55f271c072811702d92ec591c8bf86bb